modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/ImagePins.js

```diff
@@ -14,6 +14,7 @@ import PropTypes from 'prop-types';
 import React, {useLayoutEffect, useRef} from 'react';
 
 import {moveController, zoomIn, zoomOut} from './NavigationsUtils';
+const COLOR_ORANGE = '#ffa500';
```

Comment:
There needs to be an empty line between import statements and the rest of the file.